src/plugins/utils/is-response.js

```diff
@@ -0,0 +1,35 @@
+module.exports = (path, isOAS3 = true) => {
+  const operations = [
+    'get',
+    'put',
+    'post',
+    'delete',
+    'options',
+    'head',
+    'patch',
+    'trace'
+  ];
+
+  const pathLength = path.length;
+
+  // a necessary but not sufficient check for a response object
+  // without these, a schema property named "responses"
+  // may be validated as a response
+  const inResponsesSection = path[pathLength - 1] === 'responses';
```

Comment:
I think this name is misleading.  It would be better as `isResponsesProperty` or `isNamedResponses`.